internal/services/movies_service.go

```diff
@@ -80,6 +81,7 @@ func (s *MoviesService) Create(param models.CreateMovieParam) (*models.Movie, er
 
 func (s *MoviesService) Update(id uuid.UUID, param models.UpdateMovieParam) (*models.Movie, error) {
 	dbParam := db.UpdateMovieParams{
+		ID:        pgtype.UUID{Bytes: id, Valid: true},
```

Comment:
we need to create a function for this as well `ToUUID` in `pgtype_util` and use it like the ones below